documentation/vscode_and_bolt.md

```diff
@@ -0,0 +1,74 @@
+# Setting up VS Code for Bolt
+
+Bolt configuration and inventory files are written in YAML that conforms to a
+particular specification. It can speed up the development cycle and reduce
+errors if your development environment knows about these specifications, because
+you can get real-time validation and data type checking as you write.
+
+## Validating configuration and inventory files with JSON schemas
+
+Bolt configuration and inventory files are written in YAML that conforms to a
```

Comment:
I think this was my edit to the first paragraph, and my edit below is for this paragraph.

Reply:
```
You can validate Bolt's configuration and inventory files using JSON schemas
provided through the [Puppet Forge](https://forge.puppet.com). Configuring Visual 
Studio Code (VS Code) to use Bolt's JSON schemas enables VS Code's live IntelliSense 
feature, which gives you functionality like code completion and parameter info for 
available fields in the files.
```